src/isomorphic/children/__tests__/onlyChild-test.js

```diff
@@ -94,4 +94,12 @@ describe('onlyChild', () => {
     }).not.toThrow();
   });
 
+  it('should return the only child', () => {
+    var instance =
```

Comment:
Oh, whoops.
This is actually from an earlier PR that's still pending. Is there a good way to handle something like this?